src/NuGet.Clients/NuGet.PackageManagement.UI/Resources/Brushes.cs

```diff
@@ -258,5 +259,23 @@ public static void LoadVsBrushes()
             ListItemBackgroundSelectedColorKey = CommonDocumentColors.ListItemBackgroundSelectedColorKey;
             ListItemTextSelectedColorKey = CommonDocumentColors.ListItemTextSelectedColorKey;
         }
+
+        private static bool IsBackgroundColorFlightEnabled() =>
+            ExperimentationService.Default.IsCachedFlightEnabled(ExperimentationConstants.FlightFlags.PackageManagerBackgroundColor)
+            || IsForceBackgroundColorFlightEnabled();
+
+        private static bool IsForceBackgroundColorFlightEnabled()
```

Comment:
I think it is a good time for us to design NuGet experimentation API just in case if we enable more features in the future.

Reply:
Agreed. Maybe this first PR is okay with the additional method, but certainly a follow-up PR or even Spec would be good for abstracting this before we start adding more experiments.

Reply:
That's definitely the plan for future evolution of NuGet experimentation -- to build a reusable infrastructure to organize the experimentation code, make it easier to implement experiments with reusable components and to put those components under test. This PR introduces one tiny step in that direction with the constants file, but the next time we add an experiment we'll want to evolve the experimentation infrastructure even further.

Reply:
The way we handle that, is have you write an issue describing what you said, and link it here to resolve the comments.

Reply:
@kartheekp-ms one thing I'd like to add to this, in agile software development, we don't want to write code for the just in case scenario. As we enable more experiments, the API / infrastructure for experimentation will begin to emerge from the scenarios we encounter and code for. Trying to build an infrastructure too early would be programming for what we believe we will need in the future and not for what we do need. Instead, we'll build it iteratively, starting with the basic requirements and building upon it as requirements become more clear. Along the way, we can spec and design small portions to continue iterating on said infrastructure. Feel free to schedule some time with me offline to discuss this topic further.

Reply:
@jebriede think we were writing at the same time. Please create the follow-up issue noted above :)

Reply:
@jebriede - thank you for the details. I 100% agree with your comments. Feel free to resolve this conversation after addressing Donnie's comment.

Reply:
https://github.com/nuget/client.engineering/issues/854